src/layouts/Home/components/News.tsx

```diff
@@ -47,8 +47,10 @@ export const News: React.FC = () => {
         <Grid gutters="lg">
           <Row>
             <Col
+              size={12}
```

Comment:
Did this affect the layout? By default this column should fill its full-width (12 by default)

Reply:
It doesn't, good spot, fixed